src/internal/_xdropRepeatsWith.js

```diff
@@ -0,0 +1,22 @@
+var _curry2 = require('./_curry2');
+var last = require('../last');
+
+
+module.exports = (function() {
+  function XDropRepeatsWith(pred, xf) {
+    this.xf = xf;
+    this.pred = pred;
+  }
+  XDropRepeatsWith.prototype['@@transducer/init'] = function() {
+    return this.xf.init();
```

Comment:
note to self: since the `ini` functions are almost always the same, we should should just make them all point to the same function